src/main/java/io/gravitee/common/security/CertificateUtils.java

```diff
@@ -39,6 +40,18 @@
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public class CertificateUtils {
 
+    private static final Pattern BASE64_PATTERN = Pattern.compile("^[A-Za-z0-9+/=]+$");
+
+    public static boolean isBase64FlattenedCertBody(String certHeaderValue) {
+        return (
+            certHeaderValue.length() > 100 &&
+            (certHeaderValue.length() % 4 == 0) &&
+            BASE64_PATTERN.matcher(certHeaderValue).matches() &&
+            !certHeaderValue.contains("BEGIN CERTIFICATE") &&
+            !certHeaderValue.contains("END CERTIFICATE")
```

Comment:
If it matches the pattern, it means it cannot contain BEGIN CERTIFICATE and END CERTIFICATE, right?